config/locales/server.en.yml

```diff
@@ -11,6 +11,7 @@ en:
     chat_auto_silence_from_flags_duration: "Number of minutes that users will be silenced for when they are automatically silenced due to flagged chat messages."
     chat_default_channel_id: "The chat channel that will be opened by default when a user has no unread messages or mentions in other channels."
     chat_duplicate_message_sensitivity: "The likelihood that a duplicate message by the same sender will be blocked in a short period. Decimal number between 0 and 1.0, with 1.0 being the highest setting (blocks messages more frequently in a shorter amount of time). Set to `0` to allow duplicate messages."
+    chat_archive_destination_topic_status: "The status that the destination topic should be once a channel archive is completed."
```

Comment:
Just to check, we **do** want this much flexibility?

Trying to think of cases where a this site setting will cause some behaviour we don't want as opposed to something done manually.

Like setting is "closed", and someone says on chat "let's take this chat to the existing topic". But archiving the chat would cause the existing topic to close..

Reply:
> someone says on chat "let's take this chat to the existing topic"

Hmm I don't think generally you'd be archiving the whole channel if someone said that. You can just quote the relevant messages into the existing topic.

I also think in general archiving into an existing topic is kind of weird, but that's a different issue really.